test-suite/js/src/test/scala/org/scalajs/testsuite/library/ObjectTest.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Test
+import org.scalajs.testsuite.utils.Platform
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.{ JSBracketAccess, JSName }
+
+class ObjectTest {
+  import ObjectTest._
+
+  @Test def getOwnPropertyDescriptors(): Unit = {
+    val obj1 = new js.Object {
+      val property1 = 42
+    }
+    val propertyDescriptors = js.Object.getOwnPropertyDescriptors(obj1)
+    val property = propertyDescriptors("property1")
+    assertTrue(property.writable)
+    assertEquals(property.value, 42)
+  }
+
+  /* These need to be lazy vals, so that they do not blow up if there is no
+   * symbol support at all.
+   */
+  private val symA: js.Symbol = js.Symbol("a")
+  private val symB: js.Symbol = js.Symbol.forKey("b")
+
+  @Test def getOwnPropertySymbols(): Unit = {
+    val obj = (new js.Object()).asInstanceOf[ObjectCreator]
+    obj(symA) = "localSymbol"
+    obj(symB) = "globalSymbol"
+    val objectSymbols = js.Object.getOwnPropertySymbols(obj)
+    assertEquals(objectSymbols.toSeq, Seq(symA, symB))
```

Comment:
Prefer using `assertArrayEquals`, converting the JS arrays to Scala arrays. `assertArrayEquals` offers better error messages.

Reply:
This is weird.
```
 (expecteds: Array[AnyRef],actuals: Array[AnyRef])Unit
cannot be applied to (Array[scala.scalajs.js.Symbol], Array[scala.scalajs.js.Symbol])
```
though `js.Symbol` extends `scala.AnyRef`.

Reply:
You should be able to use
```scala
assertArrayEquals(Array[AnyRef](symA, symB), objectSymbols.toArray[AnyRef])
```
(explicitly specifying the right type)

Reply:
Fixed in https://github.com/scala-js/scala-js/pull/3869/commits/524b31322f0b749b8be118bfcb65140b8d167f38